Sources/Starknet/Data/Messages.swift

```diff
@@ -1,11 +1,34 @@
 import Foundation
 
 public struct MessageToL1: Decodable, Equatable {
+    public let fromAddress: Felt
     public let toAddress: Felt
-    public let payload: Felt
+    public let payload: [Felt]
 
     enum CodingKeys: String, CodingKey {
+        case fromAddress = "from_address"
         case toAddress = "to_address"
         case payload
     }
 }
+
+public struct MessageFromL1: Codable, Equatable {
+    public let fromAddress: Felt
```

Comment:
I see that in the spec, there's different type for ETH_ADDRESS, with fixed length to 40 digits. But I'm not sure how strict we should be about this.

Reply:
We decided not to be strict about it in starknet.jvm, so I assume we should follow similar approach here.